build.sh

```diff
@@ -57,12 +57,83 @@ do
   esac
 done
 
+function chooseRepo {
+  echo
+  echo "Where you would like to clone the HTML source from?"
+  echo
+  select nameorurl in "Github username" "Clone URL"; do
```

Comment:
Maybe "a GitHub fork" instead of "GitHub username"? And "a custom URL" instead of "Clone URL"?

'Where would you like to clone the HTML source from? GitHub username' seems like a weird exchange.